docs/sources/write/_index.md

```diff
@@ -1,29 +1,31 @@
 ---
-title: Write documentation
-menuTitle: Write
-description: Learn about Grafana Labs' technical documentation writing guidelines.
-weight: 400
 aliases:
   - /docs/writers-toolkit/writing-guide/
   - /docs/writers-toolkit/write/
-Keywords:
+date: 2024-02-29
+description: Learn about Grafana Labs' technical documentation writing guidelines.
+keywords:
   - writing
   - guidelines
+menuTitle: Write
+title: Write documentation
+weight: 400
 ---
 
 # Write documentation
 
-This writing guide defines the structured authoring environment we use to create documentation at Grafana Labs.
+This writing guide defines how Grafana Labs writes documentation.
```

Comment:
I felt that the previous language was overly complicated. I'm also not sure that the reader will know (or care) what "structured authoring" is.